app/controllers/users_controller.rb

```diff
@@ -1,10 +1,19 @@
 class UsersController < ApplicationController
+  before_action :load_user, only: [:show, :edit, :update]
```

Comment:
```suggestion
  before_action :load_user, only: %i(show edit update)
```
tương tự ở dưới luôn nhé em